### PR TITLE
added dummy X reg # 15

### PR DESCRIPTION
**Pull request by vviers**
solves #15,

I kept it simple

### CODE REVIEW

File: inst/tutorials/chapter4/reg_app_standalone.Rmd [hunk 1]
```diff
@@ -441,3 +441,96 @@ output$MSE2 <- renderText({
...
+    rect(xleft = x, ybottom = y,
+         xright = x + abs(errors), ytop = y + errors, density = -1,
+         col = rgb(red = 0, green = 1, blue = 0, alpha = 0.05), border = NA)
+  } else {
+    curve(expr =expr , from = min(x)-10, to = max(x)+10, add = TRUE, col = "black")
+    segments(x0 = x, y0 = y, x1 = x, y1 = (y + errors), col = "red")
+    rect(xleft = x, ybottom = y,
+         xright = x + abs(errors), ytop = y + errors, density = -1,
+         col = rgb(red = 1, green = 0, blue = 0, alpha = 0.05), border = NA)
+  }
```
Comment (floswald) [inline inst/tutorials/chapter4/reg_app_standalone.Rmd]: it shows a warning
```
Warning in plot.xy(xy.coords(x, y), type = type, ...) :
  "add" is not a graphical parameter
```

File: inst/tutorials/chapter4/reg_app_standalone.Rmd [hunk 2]
```diff
@@ -441,3 +441,96 @@ output$MSE2 <- renderText({
...
 ```
+
+## Fit of $Y$ onto $X \in \{0, 1\}$ (and a constant)
+
+```{r, echo=FALSE}
+shinyUI(fluidPage(
+  sidebarPanel(sliderInput("i_dummy", "Intercept", min = 0,
+                           max = 5, step = .25, value = 5),
+               sliderInput("s_dummy", "Slope", min = -1,
+                           max = 5, step = .25, value = .25),
```
Comment (floswald) [inline inst/tutorials/chapter4/reg_app_standalone.Rmd]: start at a better slope or it's hard to get started. 3?

**Comment by vviers** [comment]
Ok, will do

**Comment by vviers** [comment]
Done, will push this pm.

**Pull request merged**